### ISSUE TITLE
Monaco-editor v.20.0 automaticLayout not working as it did in v.19.3

### ISSUE THREAD

**Issue by sstchur** [microsoft/monaco-editor#2057]
**monaco-editor version:** 0.20.0
**Browser:** All major browsers latest version (Chromium-based, including Edge(ium) and Chrome, Webkit-based, and Firefox all exhibit same issue)
**OS:** Running on Windows 10, but I highly doubt this matters
**Playground code that reproduces the issue:**

**Description of issue:**
There appears to be a regression in how .automaticLayout works in monaco-editor v.20.0 vs how it worked in v.19.3.  In the latter, it would always resize itself to its parent container.  But in v.20.0, this doesn't always happen.

**Repro**
The playgrounds are not a good way to illustrate this issue.  Instead, I have a link to a github:  https://github.com/sstchur/tmp
1. Clone the repo
2. run npm install from the my-app directory
3. execute: npm run start

**Observe**
When resizing the react-split-pane, the monaco editor does not resize itself to its container, despite automaticLayout set to true.  If you edit package.json and change monaco-editor to version .19.3 and try again, it will work.

**Live example**
If you do not want to clone the repo, you can see a live example of both v.19.3 and v.20.0:
http://stchur.com/monacov.19.3
http://stchur.com/monacov.20.0

Note that monaco will resize itself **if** you resize the browser window, but not if you use the react-split-pane control to resize it.  However, I do not think this is related to react-split-pane.  You can remove this component and simply write some Javascript to attempt to resize the container of monaco editor, and it still will not resize properly.

Calling editor .layout() when the containing element changes size does work, but I thought automaticLayout was supposed to handle this for me?

This seems like a regression from .19.3?  If I am doing something wrong, please enlighten me as to how to fix it, and I will gladly close the issue.

**Comment by spahnke** [comment]
Duplicate of #1878

**Comment by sstchur** [comment]
@spahnke Could you help me understand?  I see that it's a dup, and I've read through the thread for #1878, but I'm confused.  It indicates that a commit was merged to master in late March, claiming to fix #1878.  But I  have to assume that what's in .20.0 is well behind master then, since this isn't fixed in .20.0?  How can I get a sense of when the latest from master will make into the next official release?

**Comment by spahnke** [comment]
Yeah, unfortunately there hasn't been a new version for a long time and especially not one since the fix was merged. It should land in the next release (0.21.0) though.

**Comment by paranoidjk** [comment]
@spahnke is there any roadmap for 0.21.0 ?

**Comment by spahnke** [comment]
Not that I know of

**Comment by rcjsuen** [comment]
> @spahnke is there any roadmap for 0.21.0 ?

Adding your thumbs up to #2074 is all we can hope for I think. :|

**Comment by alexdima** [comment]
Dup of https://github.com/microsoft/monaco-editor/issues/1855
